packages/manager/src/layout/UserStatus.tsx

```diff
@@ -34,7 +33,7 @@ export default () => {
     return <Text component={TextVariants.p}>Not Authenticated</Text>;
   }
 
-  const token = getUserToken();
+  const token = keycloak.tokenParsed as any;
```

Comment:
The interface for the token is available. 
```suggestion
  const token = keycloak.tokenParsed as ISPAshipJWT;
```

Reply:
The reason why I'm not use it because there will be a case that we use mock keycloak file.

I'm plan to move `ISPAshipJWT` to a single model file